org.aardappel.TreeSheets.yml

```diff
@@ -0,0 +1,26 @@
+app-id: org.aardappel.TreeSheets
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

Comment:
runtime is 23.08

Reply:
@hfiguiere Thanks for your feedback. Please update https://docs.flatpak.org/en/latest/first-build.html then so that this mistakes can be prevented.

Reply:
flathub has different requirements.

Reply:
> flathub has different requirements.

Ah sorry, I agree.